src/consensus/malachite/spawn.rs

```diff
@@ -202,6 +202,7 @@ impl MalachiteConsensusActors {
         .await?;
         let sync_config = ValueSyncConfig {
             request_timeout: config.sync_request_timeout,
+            status_update_interval: config.sync_status_update_interval,
```

Comment:
Why do we need this? Is it related to this change?

Reply:
No it's not related to this change. It's to speed up the tests. The tests that were exercising sync were taking a really long time to run. I think it's because peers weren't sending status updates often enough. This seemed to cut that down (though some of these tests are still really slow for reasons I haven't been able to figure out yet).